yt/visualization/line_plot.py

```diff
@@ -31,6 +31,12 @@
     linear_transform, \
     invalidate_plot
 
+class LineObject(object):
+    def __init__(self, start_point, end_point, ds, label=None):
```

Comment:
I think `ds` should come before `start_point`, to match other places in our API where we pass in a dataset or a data object first.